cmd/zed/relationship.go

```diff
@@ -70,25 +77,121 @@ var readCmd = &cobra.Command{
 	RunE:              readRelationships,
 }
 
-func readRelationships(cmd *cobra.Command, args []string) error {
+var bulkDeleteCmd = &cobra.Command{
+	Use:               "bulk-delete <resource_type:optional_resource_id> <optional_relation> <optional_subject_type:optional_subject_id#optional_subject_relation>",
```

Comment:
`<optional_subject_type:optional_subject_id#optional_subject_relation>` should not be on the Use here and should be instead in the description of the flag

Reply:
I am also not sold on using the `#` syntax

Reply:
I also think this should just be called delete to have symmetry with the API

Reply:
We already have a delete

Reply:
The `#` is from read

Reply:
The first comment still needs to be addressed.

`<optional_subject_type:optional_subject_id#optional_subject_relation>` should be in the description of the flag and not in `Use` for positional arguments.

Reply:
Its literally a copy-paste from the read command; should I fix it there as well?

Reply:
yep!

Reply:
ok we're supporting this positional and the flag, and failing now if both are provided